lib/helpers/local-assets.js

```diff
@@ -26,19 +26,35 @@ async function localAssets(app, rootEikDirectory = process.cwd()) {
     const eik = configStore.findInDirectory(rootEikDirectory);
 
     (await eik.mappings()).forEach((mapping) => {
-        app.get(mapping.destination.url.pathname, (req, res) => {
-            if (res.set) {
-                // express
-                res.set('Access-Control-Allow-Origin', '*');
-                res.set('content-type', mapping.source.contentType);
-                fs.createReadStream(mapping.source.absolute).pipe(res);
-            } else if (res.type) {
-                // fastify
-                res.header('Access-Control-Allow-Origin', '*');
-                res.type(mapping.source.contentType);
-                res.send(fs.createReadStream(mapping.source.absolute));
-            }
-        });
+        const assetPath = mapping.destination.url.pathname;
```

Comment:
Can we call `assetPath`, `pathname` to line up with the whatwg URL spec? When I think path, I think local and when I think pathname I think URL.

Reply:
Done. I squashed the commits and added the `feat:` prefix so that this will be reflected in the Changelog.

Reply:
noice